.claude/skills/docx/scripts/accept_changes.py

```
@@ -0,0 +1,130 @@
+
+    return (
+        None,
+        f"Successfully accepted all tracked changes: {input_file} -> {output_file}",
+    )
+
+
+def _setup_libreoffice_macro(profile_dir: str) -> bool:
+    macro_dir = Path(profile_dir) / "user" / "basic" / "Standard"
+    macro_file = macro_dir / "Module1.xba"
+
+    subprocess.run(
```

Ruff (0.15.7)

[error] `subprocess` call: check for execution of untrusted input

(S603)